src/app/components/wrapper-components/local-game-wrapper/local-game-wrapper.component.ts

```diff
@@ -127,14 +127,20 @@ export class LocalGameWrapperComponent extends GameWrapper<string> implements Af
     }
 
     public async proposeAIToPlay(): Promise<void> {
+
         const currentPlayerIsHuman: boolean = await this.hasSelectedAI() === false;
         await this.setInteractive(currentPlayerIsHuman);
         if (currentPlayerIsHuman === false) {
             // It is AI's turn, let it play after a small delay
             const playingAI: MGPOptional<{ ai: AbstractAI, options: AIOptions }> = this.getPlayingAI();
             if (playingAI.isPresent()) {
                 window.setTimeout(async() => {
-                    await this.doAIMove(playingAI.get().ai, playingAI.get().options);
+                    const config: MGPOptional<RulesConfig> = await this.getConfig();
+                    const gameIsOngoing: boolean =
+                    this.gameComponent.rules.getGameStatus(this.gameComponent.node, config) === GameStatus.ONGOING;
```

Comment:
indentation du cul

Reply:
+ créer un ticket "LocalGWC: proposeAIToPlay improvement: have only one timer instead of one per AI" avec une explication un peu plus détaillée du problème ?

Reply:
LGWC: proposeAIToPlay improvement: have only one timer instead of one per AI crée avec la description
If user is quick enough to select first ai player (and level) then second ai player (and level) in less than 1500ms (or AI_TIMEOUT time) then before the first timeout of player zero is called, a new one is created (inside proposeAIToPlay) which make ai not wait 1500ms between each other but make a chaos that make one of the two player seem to play quicker than the other